build/package/postremove.sh

```diff
@@ -6,7 +6,7 @@ UNIT='fw-id-agent.service'
 case "$1" in
   'remove')
     if [ -d /run/systemd/system ] ; then
-      systemctl --user daemon-reload >/dev/null || true
+      systemctl --global daemon-reload >/dev/null || true
```

Comment:
nit-pick: looking at the docs, it could be sufficient to call `systemctl daemon-reload`.

Reply:
This is the question, what is right here... does `systemctl --global daemon-reload` anything other than `systemctl daemon-reload`?

Reply:
Looking at https://github.com/systemd/systemd/blob/main/src/systemctl/systemctl-daemon-reload.c#L16 and https://github.com/systemd/systemd/commit/d77d42ed3ae95ee035dce4707777b077d1a9bf8b it seems like `--global` is wrong and will produce an error in a future systemd version.

Reply:
I removed every flag. Hopefully usage without any flag is right.